docs/guides/continuous-integration/gitlab-ci.mdx

```diff
@@ -276,6 +276,5 @@ A complete CI workflow against multiple browsers, viewports and operating
 systems is available in the Cypress Real World App (RWA).
 
 Clone the <Icon name="github" inline="true" contentType="rwa" /> and refer to
-the
-[.gitlab-ci.yml](https://github.com/cypress-io/cypress-realworld-app/blob/develop/.gitlab-ci.yml)
+the [.gitlab-ci.yml](https://github.com/cypress-io/cypress-realworld-app/blob/develop/.gitlab-ci.yml)
```

Comment:
The GitLab example has been deactivated. See also PR https://github.com/cypress-io/cypress-realworld-app/pull/1365.

Reply:
Hadn't seen that. I'll remove this ref. Thanks @MikeMcC399